src/plugins/cloudinary-analytics/index.js

```diff
@@ -1,19 +1,24 @@
-import { trackVideoPlayer } from '../../components/cld-analytics/cld-analytics';
+import connectCloudinaryAnalytics from 'cloudinary-video-analytics';
```

Comment:
i think it is better to make our package export a named entity, wdyt? 
i mean defaults are problematic, tomorrow if you decide to export another thing it must be named, and so this forces a mixed syntax for the default and named when both exporting and importing. Also this will make naming more consistent.

Reply:
I agree but I it should be done in another task - https://cloudinary.atlassian.net/browse/ME-13442